examples/repl/cmdline.rs

```diff
@@ -185,14 +186,19 @@ async fn log_msg(context: &Context, prefix: impl AsRef<str>, msg: &Message) {
     let temp2 = dc_timestamp_to_str(msg.get_timestamp());
     let msgtext = msg.get_text();
     println!(
-        "{}{}{}{}: {} (Contact#{}): {} {}{}{}{}{} [{}]",
+        "{}{}{}{}: {} (Contact#{}): {} {}{}{}{}{}{} [{}]",
         prefix.as_ref(),
         msg.get_id(),
         if msg.get_showpadlock() { "🔒" } else { "" },
         if msg.has_location() { "📍" } else { "" },
         &contact_name,
         contact_id,
         msgtext.unwrap_or_default(),
+        if msg.is_mime_modified() {
+            "[MIME-MODIFIED]️"
```

Comment:
`[HTML]`?

Reply:
could also be a very long text message if I understood @r10s correctly

Reply:
> could also be a very long text message if I understood @r10s correctly

yes, but that would be also returned as html. so, maybe [HAS-HTML] - that is also similar to `dc_msg_has_html()` suggested elsewhere.